### PR TITLE
adding timestamp to updates

### PR DESCRIPTION
**Pull request by j2kan**
decided to add timestamp to the course table so it's a more accurate representation of when it was inserted into the db and that it would have actually happened or the time wouldn't update.

### CODE REVIEW

**Comment by smwatts** [comment]
Do we want to set defaults to avoid this situation? Or should it not matter

screen shot 2018-02-11 at 12 18 30 pm

File: taworks/taform/views.py [hunk 1]
```diff
@@ -23,7 +23,6 @@
 from threading import Thread
 import pandas as pd
 from django.db.models import Count, Case, When, IntegerField, Avg
-import datetime
```
Comment (smwatts) [inline taworks/taform/views.py:26]: I use datetime for some of my code - do we need to remove it?

Comment (j2kan) [reply taworks/taform/views.py:26]: it was already imported further above.

**Comment by j2kan** [comment]
don't set default, 

on the server we can just hit 1 for now. we will wipe everything when we upload

**Review by smwatts — APPROVED** [review]
lgtm

**Review by amyleblond — APPROVED** [review]
cooL!

**Pull request merged**